src/components/Dashboard/TodoList.vue

```diff
@@ -36,54 +36,54 @@
       <v-expansion-panel
         class="align-items-top todo-item"
         expand-icon=""
-        v-for="(item, index) in orderBy(taskItems, 'order')"
+        v-for="(task, index) in orderBy(taskItems, 'order')"
         :key="index"
         :class="{
-          'disabled': !item.enabled,
-          'draft': isDraft(item)
+          'disabled': !task.enabled,
+          'draft': isDraft(task)
         }"
       >
         <v-expansion-panel-header class="todo-item-toggle no-dropdown">
           <div class="list-item">
             <div class="todo-label">
-              <h3 class="list-item__title">{{item.title}}</h3>
+              <h3 class="list-item__title">{{task.title}}</h3>
 
               <div class="bcorps-ar-subtitle"
-                v-if="entityFilter(EntityTypes.BCOMP) && isConfirmEnabled(item.type, item.status)"
+                v-if="entityFilter(EntityTypes.BCOMP) && isConfirmEnabled(task.type, task.status)"
               >
                 <p>Verify your Office Address and Current Directors before filing your Annual Report.</p>
                 <v-checkbox
                   class="todo-list-checkbox"
                   label="All information about the Office Addresses and Current Directors is correct."
-                  :disabled=!item.enabled
+                  :disabled=!task.enabled
                   v-model="confirmCheckbox"
                   @click.native.stop
                 />
               </div>
 
               <div class="list-item__subtitle">
-                <div v-if="entityFilter(EntityTypes.COOP) && item.subtitle" class="todo-status">
-                  <span>{{item.subtitle}}</span>
+                <div v-if="entityFilter(EntityTypes.COOP) && task.subtitle" class="todo-status">
+                  <span>{{task.subtitle}}</span>
                 </div>
 
-                <div v-if="isDraft(item)" class="todo-status">
+                <div v-if="isDraft(task)" class="todo-status">
                   <div>DRAFT</div>
                 </div>
 
-                <div v-else-if="isCorrection(item)" class="todo-status">
+                <div v-else-if="isCorrection(task)" class="todo-status">
                   <div>FILING PENDING</div>
-                  <div v-if="isCorrection(item)">
+                  <div v-if="isCorrection(task)">
                     <v-btn x-small icon class="info-btn">
                       <v-icon>mdi-comment-text</v-icon>
```

Comment:
Use icon mdi-message-reply per uxpin.

https://preview.uxpin.com/cefa7724eb7ad8538cb92f016364066b202fbec2#/pages/123078669/simulate/no-panels

Reply:
Done and done!